### PR TITLE
Update README and Remove DigitalOcean files (CU-8678yk217)

### PR DESCRIPTION
**Pull request by tmackinnon**
Fixing a couple typos in the README and removing the DigitalOcean files that are no longer being used since we have moved to AWS. 

Test Plan:
- [x] Deploy to Dev
- [x] SmokeTest 
- [x] Travis passes

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-8678yk217 Update BraveSensor README instructions](https://app.clickup.com/t/8678yk217)

File: server/README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ Before updating Production, we deploy the server to Staging and run the smoke te
 
 1. Wait for it to complete successfully
 
-1. On your local machine in the `server` directory, update the `PARTICLE_WEBHOOKAPI_KEY` in `.env` to the value in Staging
+1. On your local machine in the `server` directory, update the `PARTICLE_WEBHOOK_API_KEY` in `.env` to the value in Staging
```
Comment (trisapeace) [inline server/README.md:93]: Good catch!

File: server/README.md [hunk 2]
```diff
@@ -128,6 +128,8 @@ Before updating Production, we deploy the server to Staging and run the smoke te
 
    1. After you see the Boron receive the `spark/flash/status` `success` message, force the Boron to restart by sending "1" to the `Force_Reset` cloud function to start using the latest firmware. Note that this will produce an error, this is expected, please ignore.
 
+   1. Trigger a test alert using the BetaTest Boron device with the new firmware
```
Comment (trisapeace) [inline server/README.md:131]: Good idea to move this earlier.

File: server/README.md [hunk 3]
```diff
@@ -220,7 +221,7 @@ Before updating Production, we deploy the server to Staging and run the smoke te
 
    1. Pull the latest code for release: `git checkout main && git pull origin main`
 
-   1. Edit the Staging environment variables, if needed: `ansible-vault edit --ask-vault-pass environments/ssm_parameters/development.tf`
+   1. Edit the Development environment variables, if needed: `ansible-vault edit --ask-vault-pass environments/ssm_parameters/development.tf`
```
Comment (trisapeace) [inline server/README.md:224]: Haha, yup.

**Review by trisapeace — COMMENTED** [review]
Once you've deployed to Dev and seen the smoketests pass then it looks good to me.

**Review by micahdbak — APPROVED** [review]
Looks good! Thanks for making these changes!

**Pull request merged**